srcpkgs/gmsh/template

```diff
@@ -0,0 +1,47 @@
+# Template file for 'gmsh'
+pkgname=gmsh
+version=4.0.7
+revision=1
+wrksrc=${pkgname}-${version}-source
+build_style=cmake
+configure_args="
+ -DENABLE_NUMPY=$(vopt_if numpy ON OFF)
+ -DENABLE_ZIPPER=$(vopt_if zipper ON OFF)
+ -DENABLE_HXT=$(vopt_if hxt ON OFF)"
+hostmakedepends="gcc-fortran blas-devel lapack-devel SDL2-devel
+ $(vopt_if mesh "hdf5-devel fltk-devel") gmp-devel"
+depends="python blas-devel lapack-devel SDL2-devel
+ $(vopt_if mesh "hdf5-devel fltk-devel") gmp-devel"
+short_desc="Three-dimensional finite element mesh generator"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://gmsh.info"
+distfiles="https://gmsh.info/src/gmsh-${version}-source.tgz"
+checksum=c6572320d0ffdf7d2488e113861bc4bd9c38a29f7fc5b67957f6fbcb63fbdbd5
+
+build_options="hxt mesh numpy zipper"
+case "${XBPS_TARGET_MACHINE}" in
+	i686|x86_64) build_options_default="hxt mesh";;
+esac
+desc_option_hxt="Enable HXT library"
+desc_option_mesh="Enable mesh support (Required for GUI)"
+desc_option_numpy="Enable fullMatrix and numpy array conversion"
+desc_option_zipper="Enable zip file compression/decompression"
+
+post_install() {
+	rm -rf builddir/gmsh-$version-source/api
+}
+
+gmsh-demos_package() {
+	short_desc+=" - demo files"
+	pkg_install() {
+		vmove usr/share/doc/gmsh/demos
+	}
+}
+
+gmsh-doc_package() {
+	short_desc+=" - document files"
+	pkg_install() {
+		vmove usr/share
```

Comment:
be more specific with the vmove since if gmsh-doc comes before for any reason it will break gmsh-demos